lib/include/ultrahdr/gainmapmath.h

```diff
@@ -18,7 +18,10 @@
 #define ULTRAHDR_GAINMAPMATH_H
 
 #include <cmath>
+#include <cstring>
```

Comment:
Just out of curious why we need this?

Reply:
Dichen,
gainmapmath.cpp has memcpy which needs this.
It looks like all the headers required for cpp are included in this header and only one header is included in gainmapmath.cpp.